README.md

```diff
@@ -36,16 +36,15 @@ npm install
 
 There are a few options to change the behaviour of `hoodie-server`.
 
-option        | default                           |   description
-------------- | --------------------------------- | -------------
-path          | process.cwd()                     | Project path
```

Comment:
I couldn’t find out where it was used. I _think_ we needed it to test hoodie-server with `my-first-hoodie`, but we don’t do that anymore. Or do we need it for another reason?